frontera-cms/static/frontera-cms/css/src/template.home.css

```diff
@@ -0,0 +1,12 @@
+/* DO NOT ADD STYLES HERE; ONLY IMPORT OTHER STYLESHEETS */
+
+
+/* NOTE: This file exists so that the Portal and User Guide can load isolated header styles */
+
+/* Organize via ITCSS */
+/* SEE: https://confluence.tacc.utexas.edu/x/IAA9Cw */
+
+/* TRUMPS */
+@import url("_imports/trumps/s-home.css");
+@import url("../../../../../../taccsite_cms/static/site_cms/css/src/_imports/trumps/s-article-list.css");
+@import url("../../../../../../taccsite_cms/static/site_cms/css/src/_imports/trumps/s-article-preview.css");
```

Comment:
__Short-Term vs. Long-Term__
- _Now_, these styles depend on HTML tags from markup built with Django plugins _(thus they are "scope" a.k.a. `s-` styles, which makes them "trumps" that overwrite any existing styles)_.
- _Later_, these styles would depend on markup class names _only_ from markup built with TACC plugins _(thus they will be "component" a.k.a. `c-` styles, which makes them more easily re-usable)_.